packages/twenty-front/src/modules/object-metadata/schemas/updateObjectMetadataItemPayloadSchema.ts

```diff
@@ -0,0 +1,31 @@
+import camelCase from 'lodash.camelcase';
+import { z, ZodTypeDef } from 'zod';
+
+import { objectMetadataItemSchema } from '@/object-metadata/schemas/objectMetadataItemSchema';
+import { UpdateObjectInput } from '~/generated-metadata/graphql';
+
+type Input = Omit<UpdateObjectInput, 'nameSingular' | 'namePlural'>;
```

Comment:
declare this as type in types folder and import it in both places

Reply:
As this was moved to settings, I used `UpdateObjectInput` (generated type from graphql) in `useUpdateOneObjectMetadataItem` and `CreateObjectInput` in `useCreateOneObjectMetadataItem`. The respective `Input` types are now only used internally by the schemas in settings.